options/cf_options.cc

```diff
@@ -531,6 +531,10 @@ static std::unordered_map<std::string, OptionTypeInfo>
          {offsetof(struct MutableCFOptions, block_protection_bytes_per_key),
           OptionType::kUInt8T, OptionVerificationType::kNormal,
           OptionTypeFlags::kMutable}},
+        {"integrity_checks",
+         {offsetof(struct MutableCFOptions, integrity_checks),
+          OptionType::kBoolean, OptionVerificationType::kNormal,
+          OptionTypeFlags::kMutable}},
```

Comment:
When it is set dynamically, it applies to newly-created memtables, right? Perhaps this semantic suggests the name of the option should be scoped to memtables. But not a big deal if not, but the name should be scoped to things that will be reasonably changeable.

Reply:
>  it applies to newly-created memtables, right? 
Yes. 

In the future, if we add validations for reading from SST files, this can apply to newly-read blocks.